kinder/pkg/cluster/manager/actions/assets/kindnet.go

```diff
@@ -110,4 +130,5 @@ spec:
             type: FileOrCreate
         - name: lib-modules
           hostPath:
-            path: /lib/modules
+			path: /lib/modules
```

Comment:
It seems indentation is not correct here

Reply:
fixed.